### ISSUE TITLE
Unable to change Basic to admin claim via /claiminfo menu

### ISSUE THREAD

**Issue by i9hdkill** [MinecraftPortCentral/GriefPrevention#651]
I think thats due to playerData always null when checking here
https://github.com/MinecraftPortCentral/GriefPrevention/blob/f1352e12a2948b51f7cc096ff5f8365a2dbc9120/src/main/java/me/ryanhamshire/griefprevention/claim/GPClaim.java#L1918

Because changeClaimType method uses null as playerData when it calls validateClaimType

https://github.com/MinecraftPortCentral/GriefPrevention/blob/f1352e12a2948b51f7cc096ff5f8365a2dbc9120/src/main/java/me/ryanhamshire/griefprevention/claim/GPClaim.java#L1853

Basically, due to this, isAdmin is always false. And this results in a no permission message when trying to change basic claims to Admin claims.
